bin/paragon-scripts.js

```diff
@@ -85,11 +85,21 @@ const COMMANDS = {
         description: 'Include only source design tokens in the build.',
         defaultValue: false,
       },
+      {
+        name: '--output-references',
+        description: 'Include references in the build output.',
```

Comment:
@brian-smith-tcril Would it be helpful to include "Include **token** references in the build output"? This is getting at whether a token who's `$value` references another token should result in the CSS file output as a CSS variable or a hardcoded value. 

`--output-references` == include CSS variables for tokens with aliases to other tokens.

Reply:
Related, if that distinction of **token reference** is helpful, could rename the argument flag altogether to `--include-token-references`.

Reply:
I think I like "Include CSS variables for tokens with aliases to other tokens." as the description. I'm fine with keeping the argument as `--output-references`, but `--include-token-references` or `--output-token-references` work too.

Reply:
My concern about mentioning "CSS variables" specifically is that the long term version is that we're not just outputting CSS variables, but in the future also formatted tokens for iOS/Android, etc. where "CSS variables" doesn't make much sense.

Similarly, if a consumer were to extend the default `style-dictionary` config to export SCSS variables from the tokens (beyond the default CSS variables), the `--output-tokens-references` is referencing SCSS variables, not CSS variables.

Reply:
Good point, would just, "Include variables for tokens with aliases to other tokens." work?

Reply:
I'd be OK with that, though the slight tradeoff is that we'd be slightly moving away from style-dictionary's official language around "references".

> Include variables for tokens with references to other tokens."

🤔

Reply:
I was actually thinking that too. My main problem with "Include references in the build output." is that it doesn't explain what references mean or what said references may be referring to. I'd be happy with "Include references for tokens with aliases to other tokens in the build output."

Reply:
Sold, let's roll with that :) 

> Include references for tokens with aliases to other tokens in the build output.

Reply:
So now the only question is the argument name itself, any strong feelings between
* `--output-references`
* `--include-token-references`
* `--output-token-references`
or maybe even
* `--include-references`

Reply:
I think `--output` makes sense since it's controlling the output build files, whereas `--include` _could_ suggest including references pre-output.

I tend to prefer `--output-token-references`  the most, but don't feel too strongly.

Reply:
Sounds good to me, let's go with `--output-token-references`!

Reply:
Nice! This will definitely make more sense now 💯

Reply:
[inform] Implemented in follow-up commit: [`036f28c` (#3186)](https://github.com/openedx/paragon/pull/3186/commits/036f28cf6023eb750ee198dc010186f18101b533)